src/detector/detector.py

```diff
@@ -27,78 +27,80 @@
     CORNER_REFINEMENT_METHOD_DICTIONARY_INT_TO_TEXT, \
     CORNER_REFINEMENT_METHOD_DICTIONARY_TEXT_TO_INT, \
     DetectionParameters, \
-    DetectorStatus, \
     MarkerSnapshot, \
     MarkerCornerImagePoint
+from src.common.structures.capture_status import CaptureStatus
+from src.common.structures.marker_status import MarkerStatus
+
 import base64
+import abc
 import cv2.aruco
 import datetime
 import logging
 import numpy
 from typing import Any, Callable
 
-logger = logging.getLogger(__name__)
+from src.detector.implementations import abstract_marker_interface
```

Comment:
I think what you're trying to do is import the `AbstractMarkerInterface` class. If you want to do this, take a look at other folders. Note that they contain `__init__.py` files that import those classes. In those cases, you can treat the entire folder like its own python module, and import anything that is exposed in that file.